tensorflow/lite/kernels/floor_div.cc

```diff
@@ -123,8 +130,21 @@ TfLiteStatus Eval(TfLiteContext* context, TfLiteNode* node) {
       return EvalImpl<int32_t>(context, data->requires_broadcast, input1,
                                input2, output);
     }
+    case kTfLiteFloat32: {
+      return EvalImpl<float>(context, data->requires_broadcast, input1, input2,
+                             output);
+    }
+    case kTfLiteUInt8: {
+      return EvalImpl<uint8_t>(context, data->requires_broadcast, input1,
+                               input2, output);
+    }
+    case kTfLiteInt64: {
```

Comment:
In general we only add new types when they're explicitly needed. Unless you have an int64 model, let's leave that type out.